Gemfile.lock

```diff
@@ -47,6 +47,7 @@ GEM
       unicode-display_width (~> 1.0, >= 1.0.1)
     rubocop-migrations (0.1.2)
       rubocop (~> 0.41)
+    rubocop-minitest (0.0.1)
```

Comment:
The current release seems to be 0.2.1:

https://github.com/rubocop-hq/rubocop-minitest/releases

Reply:
hey @andyw8 👋 

Thanks for reviewing. Yes, current release is 0.2.1, but to use > 2.0, rubocop core should be updated. Can I update it?

thanks 🍻